utils/release_helper.py

```diff
@@ -0,0 +1,790 @@
+    highlights = []
+    for line in release.body.split('\r\n'):
+        if '### Important Highlights' in line:
+            continue
+        if '###' in line:
+            break
+        if line:
+            highlights.append(line)
+    return '\n'.join(highlights)
+
+
+def get_release_start_message(repo) -> str:
+    latest_release = get_latest_release(repo)
+    next_release_version = get_next_release_version(repo)
+    next_release_date = get_next_release_date(latest_release.published_at)
+    date = get_date_for_message(next_release_date)
+    branch = get_next_stabilization_branch(repo)
+
+    future_version = bump_version(next_release_version, 'minor')
+    future_release_date = get_next_release_date(next_release_date)
+    future_date = get_date_for_message(future_release_date)
+    future_stabilization_date = get_next_stabilization_date(future_release_date)
+    future_date_stabilization = get_date_for_message(future_stabilization_date)
+
+    template = f'''
+        Subject: stabilization of v{next_release_version}
+
+        Hello all,
+
+        The release of Content version {next_release_version} is scheduled for {date}.
+        As part of the release process, a stabilization branch was created.
+        Issues and PRs that were not solved were moved to the {future_version} milestone.
+
+        Any bug fixes you would like to include in release {next_release_version} should be
+        proposed for the *{branch}* and *master* branches as a measure to avoid
+        potential conflicts between these branches.
+
+        The next version, {future_version}, is scheduled to be released on {future_date},
+        with the stabilization phase starting on {future_date_stabilization}.
+
+        Regards,'''
+    return template
+
+
+def get_release_end_message(repo) -> str:
+    latest_release = get_latest_release(repo)
+    latest_release_url = latest_release.html_url
+    highlights = get_release_highlights(latest_release)
+    last_commit = get_contributors_last_commit()
+    new_contributors = get_new_contributors(last_commit)
+    released_version = get_latest_version(repo)
+
+    for asset in latest_release.get_assets():
+        if asset.content_type == 'application/x-bzip2':
+            source_tarball = asset.browser_download_url
+        elif asset.content_type == 'application/zip':
+            prebuild_zip = asset.browser_download_url
+        elif '.tar.bz2.sha512' in asset.name:
+            source_tarball_hash = asset.browser_download_url
+        else:
+            prebuild_zip_hash = asset.browser_download_url
+
+    # It would be more readable if this multiline string would be indented. However, this makes
+    # the final text weird, after the "format" substitutions, when highlights and new_contributors
+    # have multilines too. So, the readability was compromised in favor of simplicity and user
+    # experience. The user only needs to copy and paste, without removing leading spaces.
+    template = f'''
```

Comment:
No need to comment, because you can: https://docs.python.org/3/library/textwrap.html#textwrap.dedent

Moreover, you can declare the template as a readonly global variable, and then use the `format` string method to substitute the values.

Reply:
Actually, I started using the `dedent` here but it doesn't look good when a variable containing multi-lines is passed, as contextualized in the comment. I also tried other approaches but this was my preferred option in favor of simplicity and user experience.

Reply:
I don't fully understand - if you use dedent, then you can achieve consistent indentation of that multi-line string across the whole function body with the same result. But you know about it and consciously decided not to use it, so probably fine.

Reply:
Unfortunately I can't corroborate that the desired consistence can be achieved without impacting more in readability or user experience. But I would be happy to review any future PR proposing improvements.
Yes, it is this way by a reason and I deemed that this is not worth to change it for this case.